test/src/examples_complete_test.go

```diff
@@ -0,0 +1,53 @@
+package test
+
+import (
+	"testing"
+	"math/rand"
+	"time"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+var letterRunes = []rune("abcdefghijklmnopqrstuvwxyz1234567890")
+
+func RandStringRunes(n int) string {
+	rand.Seed(time.Now().UnixNano())
+	b := make([]rune, n)
+	for i := range b {
+		b[i] = letterRunes[rand.Intn(len(letterRunes))]
+	}
+	return string(b)
+}
+
+// Test the Terraform module in examples/complete using Terratest.
+func TestExamplesComplete(t *testing.T) {
+	t.Parallel()
+
+	testName := "ses-test-"+RandStringRunes(10)
+
+	terraformOptions := &terraform.Options{
+		// The path to where our Terraform code is located
+		TerraformDir: "../../examples/complete",
+		Upgrade:      true,
+		// Variables to pass to our Terraform code using -var-file options
+		VarFiles: []string{"fixtures.us-east-2.tfvars"},
```

Comment:
```suggestion
		VarFiles: []string{"fixtures.us-east-1.tfvars"},
```

we use `us-east-2` for all our tests.
But I suppose SES is not available in `us-east-2`, so you used `us-east-1`, but need to change it here as well